src/034/p034.exs

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env elixir
+defmodule Problem034 do
+  defp factorial(0), do: 1
+  defp factorial(1), do: 1
```

Comment:
This clause is not necessary when defining `factorial`.

Reply:
You're right, I just left it there for completion 😅